v22.1/reset-storage-parameter.md

```diff
@@ -0,0 +1,97 @@
+---
+title: SET (storage parameter)
+summary: SET (storage parameter) applies a storage parameter to a table or an index.
+toc: true
+docs_area: reference.sql
+---
+
+The `RESET (storage parameter)` [statement](sql-statements.html) reverts the value of a storage parameter on a table or an index to its default value.
+
+{{site.data.alerts.callout_info}}
+The `SET (storage parameter)` is a subcommand of [`ALTER TABLE`](alter-table.html) and [`ALTER INDEX`](alter-index.html).
+{{site.data.alerts.end}}
+
+## Syntax
+
+{% comment %} railroad diagrams here {% endcomment %}
```

Comment:
Missing remote include

Reply:
Yep, I'm going to include those once https://github.com/cockroachdb/cockroach/pull/83529 gets merged.